src/push/index.ts

```diff
@@ -70,11 +70,16 @@ export async function push(monitors: Monitor[], options: PushOptions) {
       const { error, message } = await body.json();
       throw formatAPIError(statusCode, error, message);
     }
-    const { failedMonitors } = await body.json();
-    if (failedMonitors.length > 0) {
-      throw formatFailedMonitors(failedMonitors);
-    }
-    done('Pushed');
+    body.on('data', async data => {
```

Comment:
Prefer doing this
```
for await (const data of body) {
  console.log('data', data)
}
```

Otherwise you would have to explicitly handle the stream related errors. `body.on('error') etc`

Reply:
Sure, to be honest we just didn't vet this code thoroughly as the focus has been on Kibana. But we are using `for await` in our API tests so it makes sense to follow the same pattern here.

Reply:
single try catch would work, we dont need any explicit handling if we move to async interables `for await loop.